src/Persistence.php

```diff
@@ -373,7 +371,7 @@ public function serializeLoadField(Field $f, $value)
     {
         try {
             // use $f->serialize = [encode_callback, decode_callback]
-            if (is_array($f->serialize) && isset($f->serialize[1]) && is_callable($t = $f->serialize[1])) {
+            if (is_array($f->serialize) && isset($f->serialize[1]) && ($t = $f->typecast[1]) instanceof \Closure) {
```

Comment:
these $t and $f I dont like but in another PR

Reply:
I am ok with the change here